handlers/geolocation/gelocation_test.go

```diff
@@ -391,25 +392,32 @@ func TestCdnRedirectHLS(t *testing.T) {
 	n := mockHandlers(t)
 	n.Config.NodeHost = closestNodeAddr
 	n.Config.CdnRedirectPrefix, _ = url.Parse("https://external-cdn.com/mist")
-	n.Config.CdnRedirectPlaybackIDs = []string{CdnRedirectedPlaybackID}
+	n.Config.CdnRedirectPlaybackPct = map[string]float64{CdnRedirectedPlaybackID: 1.0}
 
 	// this playbackID is configured to be redirected to CDN
 	requireReq(t, fmt.Sprintf("/hls/%s/index.m3u8", CdnRedirectedPlaybackID)).
 		result(n).
 		hasStatus(http.StatusTemporaryRedirect).
-		hasHeader("Location", fmt.Sprintf("http://external-cdn.com/mist/hls/video+%s/index.m3u8", CdnRedirectedPlaybackID))
+		hasHeader("Location", fmt.Sprintf("http://someurl.com.external-cdn.com/mist/hls/video+%s/index.m3u8", CdnRedirectedPlaybackID))
```

Comment:
On bunny side I'd need to setup plenty of hostnames:
```
fra-prod-catalyst-0.lp-playback.studio.redirect.lp-playback.studio
fra-prod-catalyst-1.lp-playback.studio.redirect.lp-playback.studio
fra-prod-catalyst-2.lp-playback.studio.redirect.lp-playback.studio
lon-prod-catalyst-0.lp-playback.studio.redirect.lp-playback.studio
lon-prod-catalyst-1.lp-playback.studio.redirect.lp-playback.studio
lon-prod-catalyst-2.lp-playback.studio.redirect.lp-playback.studio
...
```
and write Bunny Rules that matching these hostnames and serving content from individual catalysts